keyserver/internal/internal.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 type KeyInternalAPI struct {
-	db storage.Database
+	DB storage.Database
```

Comment:
It's not immediately obvious to me why this is exported?

Reply:
Because a different package creates the struct (`keyserver/keyserver.go`)

Reply:
Ah, yes, okay!